### PR TITLE
fix parcel build missing dependency

### PR DESCRIPTION
**Pull request by SirM2z**
This Issue https://github.com/ant-design/parcel-antd/issues/24#issue-771363493

Solve this issue by import tinycolor library

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,8 +22,9 @@
   },
   "dependencies": {
     "@ant-design/icons": "^4.3.0",
+    "@ctrl/tinycolor": "^3.3.1",
     "@pipcook/pipcook-core": "^1.1.2",
-    "@pipcook/sdk": "beta",
+    "@pipcook/sdk": "^1.2.1-6fd0361-beta.0",
```
Comment (yorkie) [inline package.json:27]: Shall we use the normal version instead of this?

**Review by yorkie — APPROVED** [review]
Looks good to me :)

**Comment by yorkie** [comment]
Thank you for this fix @SirM2z :)

**Comment by SirM2z** [comment]
> Thank you for this fix @SirM2z :)

:p

**Pull request merged**